cores/rp2040/SerialUART.cpp

```diff
@@ -365,6 +366,26 @@ SerialUART::operator bool() {
     return _running;
 }
 
+void SerialUART::setBreakReceived() {
```

Comment:
I don't think there is any reason for user code to fake push a BREAK event (it can't fake push chars in the receive buffer, for example).  There may also be issues w/taking the mutex in an IRQ while in FreeRTOS.

Could we just set the flag in the `_handleIRQ` call instead and drop this method?

Reply:
sure.  That should work.